### DISCUSSION TITLE
Elastic Fleet Agent Serve Hostname Change

### DISCUSSION THREAD

**Discussion by 8inaryMata1eao** [Security-Onion-Solutions/securityonion#12680] [Unsupported Versions]
We are currently running SO distributed (2.40.60) with 1 Manager, 2 Search Nodes and 2 Forwarders. The nodes are behind firewalls and not directly accessible to elastic agents via the IP address. When I set the FQDN I want the agents to use, the so-elastic-agent.exe packages still use *-so-mgr-01 and the IP address, never the new FQDN setting.  I attempted to change the manager hostname to the FQDN and that just broke the grid as expected.  In the screenshot below you can see the existing hostname/ip in red and the FQDN in green.  What is the best path to accomplish this goal of setting a new FQDN for the manager in the elastic agent package?   

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/80425093/a32ae765-539e-4757-838e-d37436dd5526)

**Comment by cm-ops** [comment]
https://docs.securityonion.net/en/2.4/elastic-fleet.html#custom-fqdn-url Were you still using the original agent installer package?

**Reply by 8inaryMata1eao** [reply]
Yeah Josh actually responded in another ticket that he updated that documentation to clarify for me.  I do have the correct package now as it rebuilds after the change, correctly.  My problems now are sadly related to the hosting provider and not SO.
